README.md

```diff
@@ -61,6 +61,34 @@ class TestCustomerModel(TestCase):
 
 Check out [documentation](<http://model-bakery.readthedocs.org/>) for more complete examples.
 
+## Contributing
+
+1. Fork this repo and clone it to your computer:
+
+```
+$ git clone git@github.com:YOUR_USER/model_bakery.git
+```
+
+2. Install the dev dependencies:
+
+```
+$ pip install -r dev_requirements.txt
+```
+
+3. Change the code and run your tests with:
+
+```
+$ make test
+```
+
+4. Before opening your PR, make sure your code changes respect the project's format with:
+
+```
+$ make lint
```

Comment:
We could also adopt `pre-commit`. What do you think?

Reply:
I'd actually also be up for it, this way contributors can catch some linting issues before CI and reduce frustrations from "waiting for red CI output". :)

Reply:
I think we're already covering this, right? Because we do have the [`.pre-commit-config.yaml`](https://github.com/model-bakers/model_bakery/blob/master/.pre-commit-config.yaml) in our source code and the [`pre-commit` lib](https://github.com/model-bakers/model_bakery/blob/master/dev_requirements.txt#L4) is listed as one of our development requirements. 

Should we add more instructions on this topic?

Reply:
Anyway, I've added more info on that with 69378cf1a937601f4063f0da3d566c821f31bcca